Jenkinsfile

```diff
@@ -1,3 +1,19 @@
+/*
```

Comment:
Copyright in jenkinsfile breaks build in jenkins.
Remove it. Probably find a way how to update copyright at only modified files.